### PR TITLE
Adding json for Proxifier Portable Edition

### PR DESCRIPTION
**Pull request by Nefariis**

### CODE REVIEW

**Comment by Ash258** [comment]
I wouls suggets to add `"extract_dir": "Proxifier PE"`. No need to prefix all bins and shortcuts with it's folder.

**Comment by Nefariis** [comment]
haha sorry, I'm still figuring this stuff out - it actually took me about two hours to get this far.

Give me a second and I'll update it

**Comment by Ash258** [comment]
You could go with: 

```json
{
    "version": "3.42",
    "homepage": "https://www.proxifier.com",
    "url": "https://www.proxifier.com/download/ProxifierPE.zip",
    "hash": "94b66df9789e811f54d986f3d7f5a509bca9f8e54701b4552565a61696ec35ea",
    "extract_dir": "Proxifier PE",
    "bin": [
        "Proxifier.exe",
        "ProxyChecker.exe"
    ],
    "shortcuts": [
        [
            "Proxifier.exe",
            "Proxifier"
        ]
    ],
    "checkver": {
        "url": "https://www.proxifier.com/download/",
        "re": "([\\d\\.]+)</a>"
    },
    "autoupdate": {
        "url": "https://www.proxifier.com/download/ProxifierPE.zip"
    }
}

```

**Comment by Nefariis** [comment]
how does the autoupdate feature work? 

and there are a few different version numbers on the page (3.42 and 1.53) - how would `([\\d\\.]+)</a>` identify between them?

**Comment by Ash258** [comment]
Test it with `.\bin\checkver proxifier-portable -f`

Normally it's first match on page. 
Your regex will never work since you don't have any html entities there. If you inspect html code, there is nothing like your regex. Use mine in latest post, and you are ready to go.

File: proxifier-portable.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "bin": "Proxifier.exe",
+    "shortcuts": [
+        [
+            "Proxifier.exe",
+            "Proxifier PE"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.proxifier.com/download/",
+        "re": "([\\d\\.]+)</a>"
```
Comment (Ash258) [inline proxifier-portable.json]: (?sm)Portable.*changelog\">([\\d\\.]+)</a>

**Comment by Nefariis** [comment]
Anything more I can do to this?

**Pull request merged**